erpnext/stock/doctype/landed_cost_voucher/landed_cost_voucher.py

```diff
@@ -187,6 +190,11 @@ def validate_purchase_receipts(self):
 
 			details = frappe.db.get_value(d.receipt_document_type, d.receipt_document, fields, as_dict=1)
 
+			key = (d.receipt_document_type, d.receipt_document)
+			if key in seen_docs:
+				frappe.throw(_("Duplicate purchase receipt found: {0} - {1}")
+					 .format(d.receipt_document_type, d.receipt_document))
+
```

Comment:
Can do it like this too.


`
  key = (d.receipt_document_type, d.receipt_document)
  if key in seen_docs or seen_doc.add(key):
    frappe.throw(_("Duplicate purchase receipt found: {0} - {1}")
		    .format(d.receipt_document_type, d.receipt_document))
`